staging/src/k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/typed/apiextensions/v1beta1/customresourcedefinition.go

```diff
@@ -127,8 +127,13 @@ func (c *customResourceDefinitions) Watch(opts v1.ListOptions) watch.AggregatedW
 func (c *customResourceDefinitions) Create(customResourceDefinition *v1beta1.CustomResourceDefinition) (result *v1beta1.CustomResourceDefinition, err error) {
 	result = &v1beta1.CustomResourceDefinition{}
 
+	objectTenant := customResourceDefinition.ObjectMeta.Tenant
```

Comment:
This file change was caused by Chenqian's none finished make update. I already updated it with my previous check in.

Reply:
I will pick up the latest change.